src/main/java/com/kodilla/ecommercee/domain/order/Order.java

```diff
@@ -16,7 +16,7 @@
 @NoArgsConstructor
 @Setter
 @Entity
-@Table(name = "ORDER")
+@Table(name = "order")
```

Comment:
tu musisz zmienić wyescapeowaną nazwę, bo inaczej nie zadziała ze względu na to, że `order` to słowo kluczowe mysql - w PR od  @dkoplenski jest to zrobione, zerknij jak to zrobić